package.json

```diff
@@ -4,16 +4,18 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "postinstall": "npm run download-webviewer",
-    "download-webviewer": "npx @pdftron/webviewer-downloader --webviewerLocation client/lib",
+    "postinstall": "node tools/copy-webviewer-files.js",
     "start": "node server/serve.js"
   },
   "author": "PDFTron Systems Inc.",
   "devDependencies": {
-    "@pdftron/webviewer-downloader": "^1.0.7",
+    "@pdftron/webviewer": "^8.0.0",
     "body-parser": "^1.18.3",
     "express": "^4.16.4",
     "multer": "^1.4.1",
     "opn": "^5.4.0"
+  },
+  "dependencies": {
+    "fs-extra": "^10.0.0"
```

Comment:
I just noticed this. `fs-extra` should be a dev dependency while `@pdftron/webviewer` should probably be a dependency.